test/functional/tests/playFromTime.js

```diff
@@ -1,109 +1,51 @@
-    - for VOD stream, check to seek at a specific time that can be in a different period
-    - check playing state, and time
-    - check if playback progressing
+- load test page
+- load stream at start time ithing last period
+- check playing time
 **/
-define([
-    'intern',
-    'intern!object',
-    'intern/chai!assert',
-    'require',
-    'test/functional/tests/scripts/player',
-    'test/functional/tests/scripts/utils'
-], function(intern, registerSuite, assert, require, player, utils) {
-
-    // Suite name
-    var NAME = 'PLAYFROMTIME';
-
-    var command = null;
-
-    // Test constants
-    var PLAYING_TIMEOUT = 10; // Timeout (in sec.) for checking playing status
-    var PROGRESS_VALUE = 5; // Playback progress value (in sec.) to be checked
-    var PROGRESS_TIMEOUT = 10; // Timeout (in sec.) for checking playback progress
-    var SEEK_TIMEOUT = 10; // Timeout (in sec.) for checking seek to be completed
-    var OFFSET_TO_START = 15; // time offset (in sec.) from the beginning of the period
-
-    var loadAtRValue = function(stream) {
-        registerSuite({
-            name: utils.testName(NAME, stream),
-
-            load: function() {
-                if (!stream.available) this.skip();
-                utils.log(NAME, 'Load stream');
-                command = this.remote.get(require.toUrl(intern.config.testPage));
-                if (!stream.dynamic) {
-                    let timeToStart = stream.periods[stream.periods.length - 1].start + OFFSET_TO_START;
-                    stream.url += '#t=' + timeToStart;
-                } else {
-                    if (stream.dvrWindow > 0) {
-                        let relativePosition = stream.dvrWindow / 2;
-                        stream.url += '#r=' + relativePosition;
-                    }
+const intern = require('intern').default;
+const { suite, before, test, after } = intern.getPlugin('interface.tdd');
+const { assert } = intern.getPlugin('chai');
+
+const utils = require('./scripts/utils.js');
+const player = require('./scripts/player.js');
+const constants = require('./scripts/constants.js');
+
+// Suite name
+const NAME = 'PLAY_FROM_TIME';
+
+// Test constants
+const OFFSET_TO_START = 15; // time offset (in sec.) from the beginning of the period
+
+exports.register = function (stream) {
+
+    suite(utils.testName(NAME, stream), (suite) => {
+
+        before(async ({ remote }) => {
+            if (!stream.available) suite.skip();
+            utils.log(NAME, 'Load stream');
+            command = remote.get(intern.config.testPage);
+            let stream_ = stream;
+            if (!stream.dynamic) {
+                let timeToStart = stream.periods[stream.periods.length - 1].start + OFFSET_TO_START;
+                stream_.url += '#t=' + timeToStart;
+            } else {
+                if (stream.dvrWindow > 0) {
+                    let relativePosition = stream.dvrWindow / 2;
+                    stream_.url += '#r=' + relativePosition;
```

Comment:
I don't think this is supported, suggest to remove or replace it with #t:posix=now - offset

Reply:
yes, you're right, I'll fix and test